File: frontend/src/models/Multa.ts
interface Multa {
  Id: number;
  Monto: number;
  Descripcion: string;
}

interface Multa_Estudiante {
  Id: number;
  Estado: string;
  Id_multa: number;
  Id_estudiante: number;
}

interface Multa_EstudianteDTO {
  Id: number;
  Estado: string;
  Estudiante: string;
  Multa: string;
}

export type { Multa_Estudiante, Multa_EstudianteDTO, Multa };

File: backend/models/Multa.go
package models

type Multa struct {
	Id          int
	Monto       float64
	Descripcion string
}

type Multa_Estudiante struct {
	Id            int
	Estado        string
	Id_multa      int
	Id_estudiante int
}

type Multa_EstudianteDTO struct {
	Id         int
	Estado     string
	Estudiante string
	Multa      string
}
